lib/screens/forgot_password/cubit/forgot_password_cubit.dart

```diff
@@ -15,43 +15,37 @@ class ForgotPasswordCubit extends Cubit<ForgotPasswordState> {
   void sendOTP(String email, ForgotPasswordState state) async {
     try {
       emit(ForgotLoading());
-      String response = await _forgotPasswordRepository.sendOTP(email);
-      if (response == "200") {
-        emit(ForgotOTPInitial());
-      } else {
-        emit(ForgotOTPFailure(response, state));
-      }
+      await _forgotPasswordRepository.sendOTP(email);
+      emit(ForgotOTPInitial());
     } on NetworkException {
       emit(ForgotNetworkError(S.networkException, state));
+    } on ResponseException catch (e) {
+      emit(ForgotOTPFailure(e.message, state));
     }
```

Comment:
Here, and in other functions below, add a `on catch(e)` with `Log.s`. Refer other cubits how it was done there.